conf/config-default.yaml

```diff
@@ -522,6 +522,14 @@ plugin_attr:
     export_addr:
       ip: 127.0.0.1
       port: 9091
+    # ngx_var_labels:
```

Comment:
I think we don't need to highlight the `var`. Instead, we can just use the name `custom_labels`.